extensions/ricescore/src/index.tsx

```diff
@@ -0,0 +1,95 @@
+import { Form, ActionPanel, CopyToClipboardAction } from "@raycast/api";
+import { markdownTable } from "markdown-table";
+import { useState } from "react";
+
+function parseNumberText(text: string | undefined): number {
+  let t = text || "";
+  if (t.includes(",")) {
+    t = t.replace(",", ".");
+  }
+  return parseFloat(t);
+}
```

Comment:
We might need to pass decimal separator through env in the future. For now this workaround is okay.

Reply:
Having the current locale would be great for that